codecov.yml

```diff
@@ -77,12 +75,26 @@ comment:
   require_base: true # must have a base report to post
   require_head: true # must have a head report to post
 
+flag_management:
+  individual_flags:
+    - name: smart-tests
+      carryforward: true
+      carryforward_mode: "labels"
+      statuses:
+        - type: "patch"
+          enabled: false
```

Comment:
Leave a link to the doc that explains this?
It's entirely opaque to me.

Reply:
So the thing that told me YAML had issues (the changes to `project` and `patch` was [the YAML validation extension](https://marketplace.visualstudio.com/items?itemName=Codecov.codecov).

Then for the available values I looked [here](https://github.com/codecov/shared/blob/36cea1a0ec45a333fbbd0b89ae1ca3860e17e6d1/shared/validation/user_schema.py#L296C33-L296C33) (the actual schema, which is now available so you can look into as well)

As for the flag values... There are [these docs](https://docs.codecov.com/docs/flags#two-approaches-to-flag-management) that explain how flag management works. The fact that it needs to have `carryforward_mode: "labels"` it's in [the getting started guide](https://docs.codecov.com/docs/getting-started-with-ats#step-2-configure-the-codecovyml-for-automated-test-selection)

Reply:
I suggested adding those links as comments where appropriate, below.